### PR TITLE
Fix mistakes from Accessibility_inspector

### PR DESCRIPTION
**Pull request by Yongjun042**
> Issue number that this PR fixes (if any). For example: 'Fixes #987654321'

> What was wrong/why is this fix needed? (quick summary only)

 Accessibility_inspector
-some alt text are missing. add more infromation to alt text. Typo error. 

 Accessibility_inspector/Simulation
-equalize image size in table. change some words.
  Usually color blind people can't distinguish colors but they are good at distinguishing specific color.

> Anything else that could help us review it

### CODE REVIEW

File: files/en-us/tools/accessibility_inspector/index.html [hunk 1]
```diff
@@ -87,19 +87,19 @@ <h3 id="Keyboard_controls">Keyboard controls</h3>
...
 
-<p>You can print the contents of the accessibility tree to json by right-clicking on an entry in the Accessibility tab and selecting <strong>Print to JSON:</strong></p>
+<p>You can print the contents of the accessibility tree to JSON by right-clicking on an entry in the Accessibility tab and selecting <strong>Print to JSON:</strong></p>
 
 <p><strong><img alt="Print to JSON right-click menu in left panel" src="accessibility-inspector-print_tree_to_json.png" style="border-style: solid; border-width: 1px;"></strong></p>
 
 <p>When you do, you will get a new tab with the selected accessibility tree loaded into the JSON viewer:</p>
 
-<p><img alt="" src="accessibility_json.png" style="display: block; margin: 0px auto;"></p>
+<p><img alt="Accessibility tree loadid in new tab JSON viewer" src="accessibility_json.png" style="display: block; margin: 0px auto;"></p>
```
Comment (hamishwillee) [inline files/en-us/tools/accessibility_inspector/index.html:102]: loaded typo

File: files/en-us/tools/accessibility_inspector/index.html [hunk 2]
```diff
@@ -152,7 +152,7 @@ <h2 id="Notable_related_features">Notable related features</h2>
 
 <h3 id="Context_menu_options">Context menu options</h3>
 
-<p>An extra context menu option is added, both for the general context menu on the web page when right/<kbd>Ctrl</kbd> + clicking a UI feature, and the HTML pane of the page inspector when right/<kbd>Ctrl</kbd> + clicking a DOM element:</p>
+<p>An extra context menu option is added, both for the general context menu on the web page when right-clicking a UI feature, and the HTML pane of the page inspector when right-clicking a DOM element:</p>
```
Comment (hamishwillee) [inline files/en-us/tools/accessibility_inspector/index.html:155]: So before this said right-click a UI feature and press CTRL on the keyboard at the same time. Now it says "just right click on UI feature". Can you confirm this is intentional - ie CTRL is not needed?

File: files/en-us/tools/accessibility_inspector/simulation/index.html [hunk 3]
```diff
@@ -14,7 +14,7 @@
...
+<p>"Color blindness" is a bit of a misnomer, since most people with these disorders can see colors, but have diffrence for the distinctions that people with normal color vision can see; color vision deficiencies affect perception across the color spectrum, not only of specific colors like red or green. Color vision deficiencies affect about 8% of men, and 0.5% of women. The most common forms of color blindness (commonly lumped together as "red-green color blindness") affect more men than women, because they are due to a mutation in a gene in the X chromosome, which men usually have only one copy of.</p>
```
Comment (hamishwillee) [inline files/en-us/tools/accessibility_inspector/simulation/index.html:17]: Please revert this. The new text is incorrect, the original text was correct.

File: files/en-us/tools/accessibility_inspector/simulation/index.html [hunk 4]
```diff
@@ -38,28 +38,28 @@
   <tr>
    <td>None<br>
     (Choose this to return to normal display)</td>
-   <td><img alt="Colorful image of a cat's face, without modification" src="28369550088_617db0d6f2_m.jpg"></td>
+   <td><img alt="Colorful image of a cat's face paint, without modification" src="28369550088_617db0d6f2_m.jpg"></td>
   </tr>
   <tr>
    <td>Protanopia (no red)</td>
-   <td><img alt="Colorful image of a cat's face, with protanopia simulation" src="colorcat_protanopia.png"></td>
+   <td><img alt="Colorful image of a cat's face paint, with protanopia simulation" src="colorcat_protanopia.png"></td>
```
Comment (hamishwillee) [inline files/en-us/tools/accessibility_inspector/simulation/index.html:45]: Please revert this, and for the other cases as it is "less correct". This is not an image of a cat's face paint, but of a painted cat face. Either way, the fact it is painted is a bit irrelevant.

**Review by hamishwillee — CHANGES_REQUESTED** [review]
Thanks for contributing. Good to see alt added to the images, but some changes needed.

File: files/en-us/tools/accessibility_inspector/index.html (hunk 2 above)
Comment (Yongjun042) [reply files/en-us/tools/accessibility_inspector/index.html:155]: this action is for showing context menu. I ctrl-clicked web page but noting happens except left-ctrl-click make link open in new tab.

File: files/en-us/tools/accessibility_inspector/simulation/index.html (hunk 3 above)
Comment (Yongjun042) [reply files/en-us/tools/accessibility_inspector/simulation/index.html]: revert simulation/index.html file.

Comment (Yongjun042) [reply files/en-us/tools/accessibility_inspector/simulation/index.html]: > 
> 
> Please revert this. The new text is incorrect, the original text was correct.

revert simulation/index.html file.

File: files/en-us/tools/accessibility_inspector/index.html (hunk 1 above)
Comment (hamishwillee) [reply files/en-us/tools/accessibility_inspector/index.html:102]: See this ^^^. "tree loadid" should be "tree loaded"

File: files/en-us/tools/accessibility_inspector/index.html (hunk 2 above)
Comment (hamishwillee) [reply files/en-us/tools/accessibility_inspector/index.html:155]: Thanks!

File: files/en-us/tools/accessibility_inspector/index.html (hunk 1 above)
Comment (Yongjun042) [reply files/en-us/tools/accessibility_inspector/index.html]: > 
> 
> See this ^^^. "tree loadid" should be "tree loaded"

my bad

**Review by hamishwillee — APPROVED** [review]
Thanks very much for these fixes!

**Pull request merged**